lib/id3c/cli/redcap.py

```diff
@@ -361,6 +361,9 @@ def _fetch(self, content: str, parameters: Dict[str, str] = {}, *, format: str =
         }
 
         response = requests.post(self.api_url, data=data, headers=headers)
+        if response.status_code != 200:
+            LOG.debug(response.json())
+
```

Comment:
FWIW, last night I wrote something for the same reasons, but with a different approach, in 86f060a.

Reply:
Awesome! I'll drop this commit, then.